guides/main_concepts/relevancy.md

```diff
@@ -157,31 +157,18 @@ To add a rule to the existing ranking rule, you have to add the rule to the exis
 ]
 ```
 
-## Importance of the attributes
+## Attribute Ranking Order
```

Comment:
Should this sub-title be after adding your rules (like it is now) or should it be after order of the rules ?
I feel like order of the rules and attribute ranking order are more similar and thus should be closer to each other. Also maybe both title could be more consistent in the wording. Lastly, it seems normal that adding your own rule comes last as it is the last step once you know all the theory.

What do you think ? If you prefer it like this, it is ok for me.

Reply:
I think we should leave it like this for now so that all the info about ranking rules stays grouped together. Plus, I think we should aim to give Attribute Ranking Order its own page in the future, because it strikes me as an important feature that we don't want to get buried. <- Might do this as part of the upcoming restructure.

Reply:
That's a nice tutorial or guide idea "How to order your fields by importance" something like that